rust-runtime/aws-smithy-http-server/examples/pokemon-service/src/bin/pokemon-service-connect-info.rs

```diff
@@ -5,10 +5,9 @@
 
 use std::net::{IpAddr, SocketAddr};
 
-use aws_smithy_http_server::{
-    request::connect_info::ConnectInfo, request::request_id::ServerRequestId,
-    request::request_id::ServerRequestIdProviderLayer,
-};
+use aws_smithy_http_server::request::connect_info::ConnectInfo;
+#[cfg(feature = "request-id")]
```

Comment:
This looks a bit hideous in an example - is it perhaps the time to spin off these examples as their own top-level crates, so that we can switch features on for them without impacting the library?